projects/1.18/assets/botania/botania/lang/zh_cn.json

```
@@ -1716,377 +1758,389 @@
   "botania.page.bellethorne1": "$(o)每朵玫瑰都有它自己的刺$(0)。",
   "botania.entry.dreadthorne": "恐惧之刺",
   "botania.tagline.dreadthorne": "只伤害成年动物",
-  "botania.page.dreadthorne0": "$(item)恐惧之刺$(0)比起$(item)娇媚之刺$(0)稍微特殊一点，因为它，不会无差别伤害生物，只会伤害$(thing)成年$(0)动物而已。",
-  "botania.page.dreadthorne1": "$(o)不，邦德先生，你可以去死了$(0)。",
+  "botania.page.dreadthorne0": "$(item)恐惧之刺$(0)比起$(l:functional_flowers/bellethorne)$(item)娇媚之刺$(0)$(/l)稍微特殊一点，因为它不会无差别伤害生物，只会伤害$(thing)成年$(0)动物而已。",
+  "botania.page.dreadthorne1": "$(o)基本演绎法$(0)。",
   "botania.entry.heiseiDream": "平成之梦",
   "botania.tagline.heiseiDream": "让怪物互相残杀",
   "botania.page.heiseiDream0": "$(item)平成之梦$(0)会用$(thing)魔力$(0)，会唤醒附近$(thing)怪物$(0)的仇恨，让它们窝里斗。",
   "botania.page.heiseiDream1": "$(o)生于忧患，死于安乐$(0)。",
   "botania.entry.tigerseye": "虎之眼",
   "botania.tagline.tigerseye": "吓唬并赶走苦力怕",
-  "botania.page.tigerseye0": "我们都知道$(thing)苦力怕$(0)害怕$(thing)猫$(0)。$(p)$(item)虎之眼$(0)利用非常微弱的，通过散发出猫的气味，恐吓附近的$(thing)苦力怕$(0)，阻止它们爆炸。也会让它们主动逃离$(thing)玩家$(0)。",
+  "botania.page.tigerseye0": "我们都知道$(thing)苦力怕$(0)害怕$(thing)猫$(0)。$(p)$(item)虎之眼$(0)通过散发出非常微弱的猫的气味，恐吓附近的$(thing)苦力怕$(0)，阻止它们爆炸。它还会让苦力怕主动逃离$(thing)玩家$(0)。",
   "botania.page.tigerseye1": "$(o)是时候向敌人反击了$(0)。",
   "botania.entry.orechid": "凝矿兰",
   "botania.tagline.orechid": "石中取矿",
   "botania.page.orechid0": "尽管挖矿很好，我们应想些可再生的方法。$(item)凝矿兰$(0)会用（大量的）$(thing)魔力$(0)来在附近的$(item)石头$(0)方块中生成$(thing)矿物$(0)。$(p)它生成的矿物是随机的，但显然普通的比稀有的偏多。",
-  "botania.page.orechid1": "$(o)这可不是龙的便便(SAO)$(0)。",
+  "botania.page.orechid1": "$(o)不太神奇的人会说这是一个筛子$(0)。",
   "botania.entry.orechidIgnem": "炎矿兰",
   "botania.tagline.orechidIgnem": "下界岩里的下界矿",
-  "botania.page.orechidIgnem0": "$(item)凝矿兰$(0)是朵不错的花。用$(thing)魔力$(0)来产生原矿，对谁而言都是极好的。然而它却不能产生来自$(thing)下界$(0)的矿物。通过改变花瓣的配方，你解决了这个问题。现在它就能利用$(item)下界岩$(0)来产生$(thing)下界$(0)矿物。花必须在$(thing)下界$(0)才能工作",
+  "botania.page.orechidIgnem0": "$(l:functional_flowers/orechid)$(item)$(item)凝矿兰$(0)(/l)是朵不错的花。用$(thing)魔力$(0)来产生原矿，对谁而言都是极好的。然而它却不能产生来自$(thing)下界$(0)的矿石。要解决这个问题很容易：更换几个花瓣就可以得到$(item)炎矿兰$(0)。现在它就能利用$(item)下界岩$(0)来产生$(thing)下界$(0)矿石，花必须在$(thing)下界$(0)才能工作。",
   "botania.page.orechidIgnem1": "$(o)变热吧$(0)。",
   "botania.entry.fallenKanade": "天降之奏",
-  "botania.tagline.fallenKanade": "强大的小范围",
+  "botania.tagline.fallenKanade": "强大的小范围治愈",
+  "botania.page.fallenKanade0": "$(item)天降之奏$(0)是有天使属性的花，会从附近的$(thing)魔力池$(0)抽取$(thing)魔力$(0)给附近的$(thing)玩家$(0)提供生命恢复的效果。",
+  "botania.page.fallenKanade0a": "$(item)天降之奏$(0)是有天使属性的花，会从附近的$(thing)魔力池$(0)抽取$(thing)魔力$(0)给附近的$(thing)玩家$(0)提供生命恢复的效果.$(p)有趣的是，$(thing)血魔法师们$(0)已经知道了这朵花会因他们祭坛魔法的属性而失效，务必注意这一点。",
   "botania.page.fallenKanade1": "$(o)音速手刃，第四版$(0)。",
   "botania.entry.exoflame": "冶炼火",
   "botania.tagline.exoflame": "点燃并给熔炉加速",
-  "botania.page.exoflame0": "不像同类的产能花，$(item)冶炼火$(0)将$(thing)魔力$(0)用于产生热量。$(p)邻近的$(item)熔炉$(0)都会被$(item)冶炼火$(0)加热，同时提高效率，消耗一些$(thing)魔力$(0)。",
+  "botania.page.exoflame0": "不像火红莲，$(item)冶炼火$(0)将$(thing)魔力$(0)用于产生热量。$(p)邻近的$(item)熔炉$(0)都会被$(item)冶炼火$(0)加热，提高效率的同时消耗一些$(thing)魔力$(0)。",

```

`Conversely to its $(thing)Generating$(0) counterpart`
```suggestion
  "botania.page.exoflame0": "与火红莲相反，$(item)冶炼火$(0)将$(thing)魔力$(0)用于产生热量。$(p)邻近的$(item)熔炉$(0)都会被$(item)冶炼火$(0)加热，提高效率的同时消耗一些$(thing)魔力$(0)。",

```